### PR TITLE
Add route subject, create view and finish show subject function

### PR DESCRIPTION
**Pull request by baochanh1801**

### CODE REVIEW

**Comment by anhndt-0059** [comment]
ok

**Comment by baochanh1801** [comment]
ready

File: resources/css/app.css [hunk 1]
```diff
@@ -275,3 +280,14 @@ a.article:hover {
...
+.nav-title{
+    color: rgba(255, 0, 0, 0.801);
+}
+
+.card-img-top {
+    width: 100%;
+    height: 15vw;
+}
+
+
```
Comment (quoctd-0607) [inline resources/css/app.css]: remove 2 line này đi

File: routes/web.php [hunk 2]
```diff
@@ -28,6 +29,17 @@
...
+        'store' => 'admin.subject.store',
+        'show' => 'admin.subject.show',
+        'edit' => 'admin.subject.edit',
+        'update' => 'admin.subject.update',
+        'destroy' => 'admin.subject.destroy',
+    ]);
 });
 
 //trainee controller
+
```
Comment (quoctd-0607) [inline routes/web.php]: remove line này

**Comment by baochanh1801** [comment]
ready

**Pull request merged**